crates/rover-client/src/operations/persisted_queries/publish/publish_mutation.graphql

```diff
@@ -6,6 +6,9 @@ mutation PublishOperationsMutation($graphId: ID!, $listId: ID!, $operationManife
         ... on PermissionError {
           message
         }
+        ... on CannotModifyOperationBodyError {
```

Comment:
Just for posterity, a better thing to do here would be `... on Error` but graphql-client seems to disallow that :( https://github.com/graphql-rust/graphql-client/issues/454